app/views/groups/new.html.erb

```diff
@@ -0,0 +1,52 @@
+
+<%= form_with model: @group, url: groups_path, html: { class: "form-logged-in" } do |form| %>
+
+<%= link_to groups_path do %>
+  <i class="fa-solid fa-circle-chevron-left back-icon "></i>
+<% end %>
+
+
+<%# # Showing error messages for validations %> 
+  <% if @group.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(@group.errors.count, "error") %> prohibited this group from being saved:</h2>
+
+      <ul>
+        <% @group.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+     <%# # Showing error messages for failures with flash %> 
+  <% flash.each do |type, msg| %>
+  <div>
+   <p> <%= msg %> </p>
+  </div>
+<% end %>
+
+       <div class="container-login">
+        <div class="row">
+          <div class="col-25">
+              <%= form.label :name, style: "display: block" %>
+           <div class="col-75">
+             <%= form.text_field :name %>
+           </div>
+          </div><br/>
+       
+     <div>
+    <%= form.label :icon, 'Select a category:' %> <br/>
+
+    <div class= "form-control">
+        <%= form.select :icon, ["⌚", "👞 ", "👔", "🎵", "🎉", "🎄", "🎁", "🍼", "✈️",
+        "🎮", "📚", "🏥", "🏈", "🎬", "🏠", "💊", "🚇", "🚔", "🐶", "🍻", "🍔", "🍟", "🚗", "🎲", "🖥"],
+          value: @group.icon, required: true, class: "form-control" %> 
+    </div>
+  </div><br/>
+
+   <div class="row">
+    <%= form.submit "Add category", class: 'add-category' %>
```

Comment:
- [ ] Kindly make sure that error messages are displayed when the user attempt to click on this button without filling out the fields so that anyone with bad intentions cannot compromise our app.